templates/user/settings/account.tmpl

```diff
@@ -59,7 +59,7 @@
 								<div class="ui selection dropdown" tabindex="0">
 									<input name="preference" type="hidden" value="{{.EmailNotificationsPreference}}">
 									{{svg "octicon-triangle-down" 14 "dropdown icon"}}
-									<div class="text">{{$.locale.Tr "settings.email_notifications"}}</div>
+									<div class="text"></div>
```

Comment:
Why?
Because the key doesn't exist?
And if yes, shouldn't it be created? Or what is the purpose of an empy div without a meaningful styleclass?

Reply:
The key doesn't exist.

The JS will fill the `<div class=text>` with selected text finally. That's how the Fomantic UI dropdown works.